packages/zosfiles/src/api/methods/mount/Mount.ts

```diff
@@ -0,0 +1,122 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { AbstractSession, ImperativeExpect, ImperativeError } from "@zowe/imperative";
+
+import { IMountFsOptions } from "./doc/IMountFsOptions";
+import { isNullOrUndefined } from "util";
+import { ZosmfRestClient } from "../../../../../rest";
+import { ZosFilesConstants } from "../../constants/ZosFiles.constants";
+import { ZosFilesMessages } from "../../constants/ZosFiles.messages";
+import { IZosFilesResponse } from "../../doc/IZosFilesResponse";
+
+/**
+ * This class holds helper functions that are used to mount file systems through the z/OS MF APIs
+ */
+export class Mount {
+    /**
+     * Mount a Unix file system
+     *
+     * @param {AbstractSession}  session         - z/OS MF connection info
+     * @param {string}           fileSystemName  - contains the file system name
+     * @param {string}           mountPoint      - contains the mount point
+     * @param {IListOptions}     [options={}]    - contains the options to be sent
+     *
+     * @returns {Promise<IZosFilesResponse>} A response indicating the outcome of the API
+     *
+     * @throws {ImperativeError} file system name must be set
+     * @throws {Error} When the {@link ZosmfRestClient} throws an error
+     *
+     * @see https://www.ibm.com/support/knowledgecenter/SSLTBW_2.1.0/com.ibm.zos.v2r1.izua700/IZUHPINFO_API_MountUnixFile.htm
+     */
+    public static async fs(
+        session: AbstractSession,
+        fileSystemName: string,
+        mountPoint: string,
+        options?: Partial<IMountFsOptions>)
+        : Promise<IZosFilesResponse> {
+        // We require the file system name and mount point
+        ImperativeExpect.toNotBeNullOrUndefined(fileSystemName, ZosFilesMessages.missingFileSystemName.message);
+        ImperativeExpect.toNotBeEqual(fileSystemName, "", ZosFilesMessages.missingFileSystemName.message);
```

Comment:
This is probably fine, but I think there is something along the lines of `toNotBeBlank`... it might also trim the value and check (to account for all spaces passed)?

Reply:
Replaced these two functions with single call to `ImperativeExpect.toBeDefinedAndNonBlank`